index.html

```diff
@@ -1,37 +1,192 @@
+        </nav>
+    </header>
+    <section class="maintext">
+        <h1 class="name">I'm Nabil Glad to see you!</h1>
+        <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project you need coded, don't hestiate to
+            contact me.</p>
+        <p class="touch">LET'S CONNECT</p>
     </section>
-    </nav>
-</header>
-<section class="maintext">
-<h1 class="name">I'm Nabil Glad to see you!</h1>
-<p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-<p class="touch">LET'S CONNECT</p>
-</section>
-<section class="icon">
-    <ul class="list">
-    <li><a href ="https://twitter.com/NeoNabil2"><img class="icons" src="twitter.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://www.linkedin.com/"><img class="icons" src="Linkedinicon.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://medium.com/"><img class="icons" src="medium.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://github.com/NabilHY"><img class="icons" src="github.png" alt="socialmediaicon"></a></li>
-    <li><a href ="https://angel.co/"><img class="icons" src="angellist.png" alt="socialmediaicon"></a></li>
-    </ul>
-</section>
+    <section class="icon">
+        <ul class="list">
+            <li><a href="https://twitter.com/NeoNabil2"><img class="icons" src="twitter.png" alt="socialmediaicon"></a>
+            </li>
+            <li><a href="https://www.linkedin.com/"><img class="icons" src="Linkedinicon.png" alt="socialmediaicon"></a>
+            </li>
+            <li><a href="https://medium.com/"><img class="icons" src="medium.png" alt="socialmediaicon"></a></li>
+            <li><a href="https://github.com/NabilHY"><img class="icons" src="github.png" alt="socialmediaicon"></a></li>
+            <li><a href="https://angel.co/"><img class="icons" src="angellist.png" alt="socialmediaicon"></a></li>
+        </ul>
+    </section>
+
+
+    <section class="works">
+        <section class="content">
+            <img class="snapshot" src="SnapshootPortfolio(1).png" alt="picture">
+            <p class="text">Tonic</p>
+            <section class="block">
+                <p class="text1">CANOPY</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="cards">
+                <li><a class="firstcard" href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
```

Comment:
- [ ] Congratulation on you have applied the changes your previous reviewers mentioned, however, I think there are still some crashes in your styles, see the image below to figure out what you will need to improve in the next coming part of the project, let's go ⏬ 

- [ ] Kindly adjust your styling, margin, padding and if any is fixed within your styling, please play with those to arrange this bug.
<img width="333" alt="Screen Shot 2022-02-12 at 15 12 52" src="https://user-images.githubusercontent.com/79058364/153713243-8a2c1363-4dcd-4317-a496-2d6c90589dc9.png">

- [ ] Please remove any fixed width for the body if any, this makes your page not behave accordingly with the child

<img width="334" alt="Screen Shot 2022-02-12 at 15 13 32" src="https://user-images.githubusercontent.com/79058364/153713253-b151bc18-3703-4b7a-9054-4842771373d1.png">

- [ ] This is actually how your card looks like when we inspect, kindly fix it on your next coming projects
<img width="333" alt="Screen Shot 2022-02-12 at 15 12 52" src="https://user-images.githubusercontent.com/79058364/153713328-45b7cc31-edfd-4f42-b0d6-0913a54c3fd3.png">

- [ ] One more thing, please fix this issue too, there is some extra spaces around your cards, please make sure you remove them all
<img width="321" alt="yuu" src="https://user-images.githubusercontent.com/79058364/153713374-1db34a7d-ee3e-48d8-941b-4cb46a37ddd8.png">